### DISCUSSION TITLE
animated video is not consistent

### DISCUSSION THREAD

**Discussion by yoga99x** [lllyasviel/ControlNet#239] [General]
I try to create an animation based on a video I converted to multiple png files. I manage to run the batch image to image but instead of a consistent animated character, it changes with almost every frame. How do I keep the animated character continuous? Thanks!

**Comment by marcsyp** [comment]
This is a common, unsolved problem, as the controlnet and SD have no knowledge of previous frames for coherence. There are a lot of threads exploring this topic and how to solve these problems and improve coherence (in the future), check around the discussions for more info.  For instance, there is some related conversation here: https://github.com/lllyasviel/ControlNet/discussions/30#discussioncomment-4953113

**Reply by yoga99x** [reply]
Thanks so much :)
